plugins/versionpress/tests/Unit/EntityInfoTest.php

```diff
@@ -128,7 +128,7 @@ public function ignoredEntityCapitalizedIsCorrectlyIdentified()
         $entity = [
             'some_field' => 'b',
             'other_field' => 'a',
-            'capitalized_value_field' => 'VaLuE', // comparison must be case insensitive
```

Comment:
This is how the test should look – we indeed want to test that if `VaLuE` is provided, it is still matched by a rule  that uses uppercase (or lowercase, or any-case) `VALUE`.

Reply:
Yes, we are providing `VALUE` in original `$entitySchema`, so we are also expecting `VALUE`. Comparison is not case insensitive now, so `VaLuE` would fail.